dashboard/cron.py

```diff
@@ -365,6 +369,8 @@ def update_with_bq_access(self):
             # Make resource data from resource_access data
             resource_df = resource_access_df.filter(["resource_id", "resource_type", "name"])
             resource_df = resource_df.drop_duplicates(["resource_id"])
+            # pangres.upsert() requires DataFrame to have index
+            resource_df = resource_df.set_index('resource_id')
```

Comment:
I just added this back in https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1032

Realized I must have dropped it when I resolved some conflicts

Reply:
I think we can just merge this one and close the other.

Reply:
Yeah, it wouldn't run for me without that index.